### PR TITLE
Add Common HIPAA Questions post

### PR DESCRIPTION
**Pull request by chasballew**
WIP for now, will finish and push shortly. Comments welcome.

### CODE REVIEW

File: common_HIPAA_questions.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+The HIPAA regulations apply to organizations, not products or features. A company can say they are “HIPAA-compliant”, but it doesn't make sense to refer to a product that way.
+
+If you are regulated, HIPAA requires that you ensure your organization:
+
+1. Control how you use regulated data internally and how you disclose it externally.
+1. Manage data security and risk with formal policies and internal controls.
+1. Identify and respond to security incidents and potential breaches of regulated data.
+
+That is a drastically simplified summary, of course. In practice, the [pilot audit protocol HHS](http://www.hhs.gov/hipaa/for-professionals/compliance-enforcement/audit/protocol-current/index.html) used for its first round of audits has several hundred “key activities,” most of which contain several audit procedures. The hard part is being able to prove to an auditor or OCR enforcement agent that you did everything you were required to, and that you made reasonable decisions along the way.
```
Comment (gib) [inline common_HIPAA_questions.md]: This is very clear. I love the first point

File: common_HIPAA_questions.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+# Who does HIPAA regulate?
+HIPAA’s formal name is the Health Insurance Portability and Accountability Act of 1996. Knowing that, you might not be surprised to learn that HIPAA regulates entities that handle data that has been (or will be) related to a health insurance transaction.
```
Comment (gib) [inline common_HIPAA_questions.md]: HIPAA - 20 years!

**Comment by wcpines** [comment]
@chasballew 

> Is is reasonable

Should be "Is it"

https://github.com/aptible/aptible-blog/commit/cbf5019acf91c825ac40e7478c76c45efea3e6bb#diff-a7d5246e8ce292a07ff7d89e4b35a1c6R84

**Pull request merged**